src/redux/reducers/applicationReducer.ts

```diff
@@ -72,6 +79,7 @@ interface State {
   hidePostsThumbnails: boolean;
   isTermsAccepted: boolean;
   isBiometricEnabled: boolean;
+  prevLoggedInUsers: PrevLoggedInUsers[];
```

Comment:
not sure if this the right place for `preLoggedInUsers`, for one application store has already overflown, and second since this is more related to account, perhaps it should be in `accountReducer`

<img width="923" alt="Screenshot 2024-08-13 at 15 39 30" src="https://github.com/user-attachments/assets/bb673bd2-6814-42fc-9793-0f4b3de6c3a1">